.github/workflows/formal-verifiation.yml

```diff
@@ -0,0 +1,54 @@
+name: Formal verification
+
+on:
+  push:
+    branches:
+      - master
+      - release-v*
+  pull_request: {} # just for the duration of the PR to test, must be removed before merging
+  workflow_dispatch: {}
+
+jobs:
+  list-scripts:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: echo "matrix=$(echo -n certora/scripts/*.sh | jq -Rsc 'split(" ")')" >> $GITHUB_OUTPUT
```

Comment:
So I managed to make that work without ls/grep, which the linter complains about, but now it asks me to `Double quote to prevent globbing and word splitting` when globbing is exactly what I want.